### ISSUE TITLE
call initState on each visit

### ISSUE THREAD

**Issue by vytautas-pranskunas-** [jb3rndt/PersistentBottomNavBarV2#23]
Hi,

when using original Flutter BottomTabNavigation ```initState``` is called every time when visited. This allows to refresh data upon visit for example. However I can see that this plugin acts differently - calls ```initState``` only once on first visit. Is there a way to atchieve same behaviour - call ```initState``` on each visit?

**Comment by jb3rndt** [comment]
Yes you can set `stateManagement: false` then it will be regenerated on every visit

**Comment by vytautas-pranskunas-** [comment]
@jb3rndt after small debugging i was about to write the same :)

**Comment by ermand-hoxha** [comment]
I have the same problem. If i put `stateManagement: false` all pages are rebuild, not only the screen where i tap. Is there any solution ?

**Comment by jb3rndt** [comment]
Hm I took a look into it and did not find a way to solve it yet. I'm going to open a new issue for it

**Comment by Shubham-Narkhede** [comment]
Is there any solution on this ?

**Comment by jb3rndt** [comment]
Can you try setting `animateTabTransition` to `false` like this? (In `PersistentTabView`)
```
screenTransitionAnimation: ScreenTransitionAnimation(
  animateTabTransition: false,
  curve: Curves.ease,
  duration: Duration(milliseconds: 200),
),
```
